README.md

```diff
@@ -1609,11 +1609,14 @@ The benefits of Terraform over the other tools:
 <details>
 <summary>Explain each of the following:
 
-  * Provider
-  * Resource
-  * Provisioner
-</b></details>
-
+  * Provider: 
+  * Resource: 
+  * Provisioner: 
+</b></details><br><b>
```

Comment:
Here summary should be closed in order to wrapp the question.

Reply:
Also details should not be closed here. As @bregman-arie  showed you, details wraps the entire question + answer whereas summary only wraps the question. You can check your own fork to check whether the changes are alright or not.
https://github.com/knoxknot/devops-interview-questions/blob/master/README.md#terraform-beginner